lib/ssl/src/ssl.erl

```diff
@@ -3385,10 +3433,12 @@ reading and writing keys are updated.
 %%
 %% Description: Initiate a key update.
 %%--------------------------------------------------------------------
-update_keys(#sslsocket{pid = [Pid, Sender |_]}, Type0) when is_pid(Pid) andalso
-                                                            is_pid(Sender) andalso
-                                                            (Type0 =:= write orelse
-                                                             Type0 =:= read_write) ->
+update_keys(#sslsocket{connection_handler = Pid,
```

Comment:
this PR improves readability of code in general.
but it could be further improved if `Pid` variable is replaced with something more meaningful.

not a big problem for me because:
- I know the convention and some of ssl code, so when I see `Pid` I recall it is the process not being the Sender ;-) (AKA the other but main one)
- function clauses are typically small and it is easy to glimpse where `Pid` comes from

also `is_pid(Pid)` is a bit confusing ...
maybe fine since used in a lot of places and short enough ... ignore if you want.

Reply:
Renamed it Controller (as it is the socket owner that can be changed with ssl:controlling_process).